tapir/core/tapir_email_base.py

```diff
@@ -132,8 +133,8 @@ def send_to_share_owner(self, actor: User | None, recipient: ShareOwner):
         )
 
     def user_wants_to_or_has_to_receive_mail(self, user: TapirUser):
-        return (self.get_unique_id() in user.additional_mails) | (
-            self.get_unique_id() in [x[0] for x in mails_mandatory()]
+        return (self.get_unique_id() in user.optional_mails_by_user()) | (
```

Comment:
`|` could be an `or`, right?

Reply:
yep, was working with bitwise operators a lot at that time